model/src/test/kotlin/DependencyTreeNavigatorTest.kt

```diff
@@ -222,6 +222,41 @@ class DependencyTreeNavigatorTest : WordSpec() {
             }
         }
 
+        "projectDependencies" should {
+            "return the dependencies of a project" {
+                val scopeDependencies = navigator.scopeDependencies(testProject)
+                val projectDependencies = navigator.projectDependencies(testProject)
+
+                projectDependencies should containExactlyInAnyOrder(
+                    scopeDependencies["compile"].orEmpty() + scopeDependencies["test"].orEmpty()
```

Comment:
Maybe use `getValue("compile/test")` to be on the safe side in case of unexpected changes to the test data.